src/modules/closeButton.test.js

```diff
@@ -0,0 +1,14 @@
+import { closeButton } from './closeButton.js';
+
+jest.mock('./variables.js');
+
+describe('Remove a list item', () => {
+  it('should expose a function', () => {
+    expect(closeButton).toBeDefined();
+  });
+
+  it('Close button should remove a list item', () => {
+    // const retValue = closeButton(index);
```

Comment:
Kindly remove commented lines of code to make your code more cleaner.

Reply:
Thanks!